apache_license.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
wasn't sure where to put this file

Reply:
where does it come from?

Reply:
i use it as a header for the generation of the the deepcopy functions

Reply:
I'd say the less terrible place is under dev-env with a proper name (such as deepcopy_header.txt or something like that)